tests/countries/test_bonaire_sint_eustatius_and_saba.py

```diff
@@ -0,0 +1,314 @@
+
+
+class TestBonaireSintEustatiusAndSaba(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        years = range(2011, 2050)
+        super().setUpClass(BonaireSintEustatiusAndSaba, years=years)
+        cls.subdiv_holidays = {
+            subdiv: BonaireSintEustatiusAndSaba(subdiv=subdiv, years=years)
+            for subdiv in BonaireSintEustatiusAndSaba.subdivisions
+        }
+
+    def test_country_aliases(self):
+        self.assertAliases(BonaireSintEustatiusAndSaba, BQ, BES)
+
+    def test_new_years(self):
+        name = "Nieuwjaarsdag"
+        self.assertHolidayName(name, (f"{year}-01-01" for year in range(2011, 2050)))
+
+    def test_good_friday(self):
+        name = "Goede Vrijdag"
+        dt = (
+            "2020-04-10",
+            "2021-04-02",
+            "2022-04-15",
+            "2023-04-07",
+            "2024-03-29",
+            "2025-04-18",
+        )
+        self.assertHolidayName(name, dt)
+
+    def test_easter_sunday(self):
+        name = "Eerste Paasdag"
+        dt = (
+            "2019-04-21",
+            "2020-04-12",
+            "2021-04-04",
+            "2022-04-17",
+            "2023-04-09",
+            "2024-03-31",
+            "2025-04-20",
+        )
+        self.assertHolidayName(name, dt)
+
+    def test_easter_monday(self):
+        name = "Tweede Paasdag"
+        dt = (
+            "2019-04-22",
+            "2020-04-13",
+            "2021-04-05",
+            "2022-04-18",
+            "2023-04-10",
+            "2024-04-01",
+            "2025-04-21",
+        )
+        self.assertHolidayName(name, dt)
+
+    def test_kings_day(self):
+        name_queen = "Koninginnedag"
+        name_king = "Koningsdag"
+        self.assertHolidayName(name_queen, (f"{year}-04-30" for year in range(2011, 2014)))
+        self.assertNoHolidayName(name_queen, range(2014, 2050))
+        self.assertHolidayName(
+            name_king,
+            "2014-04-26",
+            "2015-04-27",
+            "2020-04-27",
+            "2021-04-27",
+            "2022-04-27",
+            "2023-04-27",
+            "2024-04-27",
+            "2025-04-26",
+        )
+        self.assertNoHoliday(
+            "2014-04-27",
+            "2025-04-27",
+        )
+        self.assertHolidayName(name_king, range(2014, 2050))
+        self.assertNoHolidayName(name_king, range(2011, 2014))
+
+    def test_rincon_day(self):
+        name = "Rincondag"
+        self.assertNoHolidayName(name)
+
+        for subdiv, holidays in self.subdiv_holidays.items():
+            if subdiv == "BON":
+                self.assertHolidayName(
+                    name,
+                    holidays,
+                    "2020-04-30",
+                    "2021-04-30",
+                    "2022-04-30",
+                    "2023-04-30",
+                    "2024-04-30",
+                    "2025-04-30",
+                )
+                self.assertHolidayName(name, holidays, range(2020, 2050))
```

Comment:
```suggestion
                self.assertHolidayName(name, holidays, (f"{year}-04-30" for year in range(2020, 2050)))
                self.assertNoHolidayName(name, holidays, range(2011, 2020))
```
And same for similar fixed date holidays.